app/models/archived/intake_2021.rb

```diff
@@ -298,8 +298,6 @@ def self.searchable_fields
         self.email_domain = email_address.split('@').last.downcase
         self.canonical_email_address = compute_canonical_email_address
       end
-      self.primary_last_four_ssn = primary_ssn&.last(4) if encrypted_primary_ssn_changed?
-      self.spouse_last_four_ssn = spouse_ssn&.last(4) if encrypted_spouse_ssn_changed?
```

Comment:
This was added for behavior that is specific to this tax season and moving forward, but we hadn't archived last year's intake model yet. This logic caused some issues when trying to set up archived intakes in the tests without a full ssn, and since its not relevant to how last year's intakes worked, im removing it.